src/main/pages/che-7/administration-guide/con_controller-requirements.adoc

```diff
@@ -8,7 +8,7 @@ The Workspace Controller consists of a set of five services running in five dist
 [cols="2,2,1,1", options="header"]
 .ControllerServices
 |===
-|Pod
+|{prod-short} Server and Dashboard
```

Comment:
Uh.... no. It's still just a pod here. My suggestion was for line 16 below:

-|{prod-short}
+|{prod-short} Server and Dashboard